docs/source/gbd2017_models/causes/ischemic_heart_disease/index.rst

```diff
@@ -109,15 +109,15 @@ The aspects of the disease this cause model is designed to simulate are the stat
 Model Assumptions and Limitations
 +++++++++++++++++++++++++++++++++
 
-Apart from inpatient hospital and inpatient claims data, GBD 2017 did not include any data from sources other than the literature for myocardial infarction. Given this information, the assumption is that MI is the best and only estimator for the IHD model. The limitation of this assumption and approach is the exclusion of non-MI data sources could be underestimating the IHD model. 
+Apart from inpatient hospital and inpatient claims data, GBD 2017 did not include any data from sources other than the literature for myocardial infarction. Given this information, the assumption is that MI is the best and only estimator for the IHD model. The limitation of this assumption and approach is the exclusion of non-MI data sources could be underestimating the IHD model. In the Vivarium Modeling Strategy, the limitation of the IHD model is we do not initial 'Acute MI' state in the first 28 days, and assume that prevalence = 0.
 
 .. todo::
 
    Describe more assumptions and limitations of the model.
 
 Cause Model Diagram
 --------------------
-.. image:: ischemic_heart_disease_transitions.svg
+.. image:: cause_model_ihd.svg
```

Comment:
I don't think we've documented a model of this type before. Is there a epi standard way to denote this? Is SAP the standard notatoin? I did a very, very quick google of types of compartmental disease models, and didn't see this one. do you know @SantoniS @alibow ? Other people with epi knowledge?

Reply:
This was the documented model that we came up with in a meeting with @aflaxman  @collijk  and @davidbernotas. If it should be updated, let me know!

Reply:
it's not entirely different from an SIR model with a carrier state.  SI model with a carrier state, I suppose...